events/decisions_events/wc_purge_events.txt

```diff
@@ -0,0 +1,335 @@
+﻿namespace = purge
+
+## Start events
+
+#Nothing happens
+purge.0010 = {
```

Comment:
The purge should be on a separate branch from the basic Plague mechanics.
When you mix different things on one branch, you become difficult to both design and test.

Reply:
How can you test purging mechanics if you have no plagued barony?